circus-train-gcp/src/main/java/com/hotels/bdp/circustrain/gcp/context/GCPSecurity.java

```diff
@@ -32,4 +33,11 @@ public void setCredentialProvider(String credentialProvider) {
     this.credentialProvider = credentialProvider;
   }
 
+  public String getProjectId() {
```

Comment:
what is this?
if it is plugin specific config it should live in the plugin if it makes sense to have it here it needs some documentation